crates/bevy_render/src/lib.rs

```diff
@@ -198,7 +198,7 @@ impl Plugin for RenderPlugin {
                 .add_stage(
                     RenderStage::Render,
                     SystemStage::parallel()
-                        .with_system(PipelineCache::process_pipeline_queue_system)
+                        .with_system(PipelineCache::process_pipeline_queue_system.at_start())
```

Comment:
Was this strictly necessary for this PR? Probably makes sense if we ever add other systems to the stage (i.e. command buffer/render bundle based parallelization).

Reply:
I didn't try, but I believe we do need to make sure this runs before any of the render nodes. Otherwise, you potentially won't be able to access a RenderPipeline that you inserted in the same frame depending on system execution order.

Reply:
Oh yeah that makes sense. I removed the `at_start` and left a comment instead, in case we move things around in future.